config/application.rb

```diff
@@ -23,6 +23,9 @@ class Application < Rails::Application
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
 
+    # Enable deflate / gzip compression of controller-generated responses
+    config.middleware.use Rack::Deflater
```

Comment:
Good point, it could... From other other view the goal was to have the same behavior in all environments.
I will mesure time before and after this change.